packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+
+<ApiOperation method="get" url="/groups" />
+
+When importing groups from the SCIM server, Okta accesses the `/groups` endpoint and processes them page by page, using `startIndex`, `count` and `totalResults` values for reference.
+
+Okta always uses `count=100` as reference to return 100 elements. If the value of `totalResults` is higher than 100, then, after finishing retrieving the resources, `startIndex` becomes `startIndex+100` and it is passed as a query parameter along with `count` in a new request to the `/groups` endpoint. This pagination operation repeats until all pages are viewed. The number of pages is calculated using the formula:
+
+    pages = ceil(totalResults / 100)
+
+A sample request from Okta to retrieve the groups:
+
+    GET /scim/v1/groups?startIndex=1&count=100 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+The response to this request:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:07:12 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "totalResults": 1,
+        "startIndex": 0,
+        "itemsPerPage": 1,
+        "Resources": [{
+            "schemas": ["urn:scim:schemas:core:1.0"],
+            "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+            "displayName": "Test SCIM2",
+            "members": [],
+            "meta": {
+                "created": "2019-10-18T07:02:44+00:00",
+                "lastModified": "2019-10-18T07:02:44+00:00",
+                "version": "12ba4e7e158aa0b78d78c7e9ae15d917"
+            }
+        }]
+    }
+
+**Note:** The `itemsPerPage`, `startIndex`, and `totalResults` values need to be returned as integers and not as strings.
+
+### Retrieving specific groups
+
+<ApiOperation method="get" url="/groups/${groupId}" />
+
+There are situations where Okta needs to run a GET operation on a specific `${groupId}`, for example to see if the group still exists on the SCIM server. The request looks like the following:
+
+    GET /scim/v1/groups/74094a55-c9ee-47ae-9fd4-9137deb43497 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+The SCIM server response is to return the group details:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:08:51 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2",
+        "members": [],
+        "meta": {
+            "created": "2019-10-18T07:02:44+00:00",
+            "lastModified": "2019-10-18T07:02:44+00:00",
+            "version": "12ba4e7e158aa0b78d78c7e9ae15d917"
+        }
+    }
+
+### Update specific group name
+
+<ApiOperation method="patch" url="/groups/${groupId}" />
+
+When a group is pushed to the SCIM server, most of the time the group name is updated or inserted by Okta through a PATCH operation:
```

Comment:
When a group is pushed to the SCIM server, the group name is updated or inserted by Okta through a PATCH operation: